src/hooks/BalanceSummary.ts

```diff
@@ -0,0 +1,140 @@
+import { useMemo } from 'react'
+import { btcToSats } from '../utils'
+import { useCurrentWalletInfo, BalanceString, Utxos } from '../context/WalletContext'
+
+interface BalanceSummary {
+  totalBalance: string | null
+  /**
+   * @description available balance (total - frozen - locked); this value is incorrect for backend versions <= 0.9.6
+   */
+  availableBalanceDontUseYet: string | null
```

Comment:
So if I understand it correctly, once we switch to 0.9.7 we will use this value only and remove the custom calculation completely, right?

Reply:
Sadly, not quite yet. [As hinted by Adam](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/986#issuecomment-1128681700), this is still subject to change. `availableBalanceDontUseYet` and `calculatedAvailableBalanceInSats` can still differ, whereby the calculated value is "more correct" and needs to be used even after v0.9.7, unless the server handling is adapted.

So in short: It is still needed at the time of writing.

Reply:
Ok I see. From a gut feeling I'd say let's remove the deprecation flags for the calculated fields and also remove the `availableBalanceDontUseYet` field and fully rely on our own calculations then. We don't really know when (or if) the backend will implement this correctly I'd say we're better off (for now at least) not worrying about it and do it ourselves. What do you think?